pipeline/splicing/Makefile

```diff
@@ -0,0 +1,65 @@
+build:
+	# Build and tag the image prefixed by user to avoid conflicts on shared machines
+	docker build -t $(USER)-splicing-pipeline .
+
+push:
+	# Tag and push our current docker build to dockerhub
+	docker tag $(USER)-splicing-pipeline:latest brcachallenge/splicing-pipeline:latest
+	docker push brcachallenge/splicing-pipeline:latest
+
+debug:
+	# Run the docker mapping out of local directory for development
+	docker run --rm -it \
+		--entrypoint /bin/bash \
+		--user=`id -u`:`id -g` \
+		-v `readlink -f ~/references/splicing`:/references \
+		-v `pwd`:/app \
+		$(USER)-splicing-pipeline
+
+test:
+	# Run a short test from within the docker container
+	python calcVarPriors.py test short
+
+references:
+	# Download and install references
```

Comment:
I really like this Makefile for the ease of use!

Two details came up when I was testing it:
1. It would be good to have the host path for references (`~/references/splicing`) in a variable in the Makefile to easily change it.
2. Making sure in the `references` target that the reference host directory exists before starting the container(e.g. using sth like [[ -d "${REFERENCES_DIR}" ]] || mkdir -p "${REFERENCES_DIR}"). Otherwise they get created by the docker daemon (at least on the system I was testing it), which means they are only accessible by root...

Otherwise things worked smoothly!

Reply:
Good suggestions - added! Will issue a new pull request once I have switched over to click and simplified a few other things.